src/filters/bloomfilter.rs

```diff
@@ -207,6 +221,21 @@ where
         }
     }
 
+    /// Same as `with_existing_filter` but with specific `BuildHasher`.
+    pub fn with_existing_filter_and_hash<I: IntoIterator<Item = u32>>(
+        m: usize,
+        k: usize,
+        bitmap: I,
+        buildhasher: B,
+    ) -> Self {
+        Self {
+            bs: FixedBitSet::with_capacity_and_blocks(m, bitmap),
+            k,
+            builder: HashIterBuilder::new(m, k, buildhasher),
```

Comment:
Does this check that the iterator has the correct number of elements for the given parameters?